solaris/solaris10/preinstall.sh

```diff
@@ -3,22 +3,19 @@
 # Wazuh, Inc 2015-2020
 
 if [ ! -f /etc/ossec-init.conf ]; then
-	DIR="/var/ossec"
-	ls -la /var/ossec > /dev/null 2>&1
-    if [ -d  /var/ossec ]; then
-		#upgrade
-        type=upgrade
-
-	else
-		#clean installation
-        type=install
-    fi
-
+  DIR="/var/ossec"
+  ls -la /var/ossec > /dev/null 2>&1
+  if [ -d /var/ossec ]; then
+    #upgrade
+    type=upgrade
+  else
+    #clean installation
+    type=install
+  fi
```

Comment:
Please review tabulation, in bash we use 4 spaces to tabulate.

Reply:
We followed the existent tabulation in the below lines i.e. line 28.
We will modify the rest of the tabulations too. Regards.